### PR TITLE
Marketing page post

### PR DESCRIPTION
**Pull request by melinath**
This is just a first draft... I would be interested in getting feedback, and maybe also some help getting the styles right? Though maybe this should just wait for wagtail?

### CODE REVIEW

File: blog/_posts/2015-06-16-evolution-of-a-marketing-page.markdown [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+{% include image.html src="/media/2015/2015-06-16-2-about.png" %}
+
+This was an improvement, but it still didn't go far enough. Our next iteration moved the splash page header into a sidebar widget, effectively merging the user experience for authenticated and anonymous users. Meanwhile, the About page got an additional section on pricing and features, and standardized its call-to-actions on emailing us.
+
+{% include image.html src="/media/2015/2015-06-16-3-splash.png" %}
+{% include image.html src="/media/2015/2015-06-16-3-about.png" caption="Note how much clearer the elevator pitch is with higher contrast and one sentence per line – plus we added in a line suggesting some possible uses!" %}
+
+May 8, 2015: Incorporating illustrations & polishing copy
+
+In this stage, we incorporated a set of illustrations we'd commissioned from [Lucy Bellwood](lucybellwood.com). They added a great flair of personality to the previously text-only pages. We also put some extra work into making the pages responsive, and rewrote the copy to highlight the most important points.
```
Comment (nmorduch) [inline blog/_posts/2015-06-16-evolution-of-a-marketing-page.markdown]: Link for Luch needs http://

**Comment by nmorduch** [comment]
Have the dwolla fees changes such that they should be changed on Dancerfly and in these screenshots?

I like it though. Good post!

**Comment by melinath** [comment]
Okay, I think this is ready! If everything looks good I can update the post date and merge into master.

File: blog/_posts/2015-06-16-evolution-of-a-marketing-page.markdown [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+Third, think about what you want people coming to your site to *do*. It's generally safe to be consistent! In our case, we made the mistake of asking people to "Sign up" even though account creation wasn't actually required. "Email us" is much clearer and has proven more effective.
+
+Fourth, split important information into digestible chunks – big, lots of whitespace, and high contrast. Hire an illustrator or photographer to add some life to the words on the page!
+
+And finally – maybe this one is obvious – don't try to get it perfect the first time. Getting to where we are took over a year, and having the more primitive versions didn't stop us from getting off the ground. And as far as we've come, I'm sure this won't be the last version of our marketing pages.
+
+
+*If you'd like to use Dancerfly for a dance event **you** organize, send us an email at <hello@littleweaverweb.com>.*
```
Comment (nmorduch) [inline blog/_posts/2015-06-16-evolution-of-a-marketing-page.markdown]: Formatting typos here

**Comment by nmorduch** [comment]
:+1: after you fix the formatting in line 94

File: blog/_posts/2015-06-16-evolution-of-a-marketing-page.markdown (hunk 2 above)
Comment (melinath) [inline blog/_posts/2015-06-16-evolution-of-a-marketing-page.markdown]: nnno?

Comment (melinath) [inline blog/_posts/2015-06-16-evolution-of-a-marketing-page.markdown]: screen shot 2015-06-23 at 2 51 33 pm

Comment (nmorduch) [inline blog/_posts/2015-06-16-evolution-of-a-marketing-page.markdown]: Huh, looks like GitHub is having problems with it for no reason

**Pull request merged**